runtime/altair/src/migrations.rs

```diff
@@ -90,6 +91,9 @@ pub type UpgradeAltair1034 = (
 	runtime_common::migrations::increase_storage_version::Migration<crate::Council, 0, 4>,
 );
 
+#[cfg(feature = "std")]
+pub type UpgradeAltair1034 = ();
+
```

Comment:
Maybe there exists a way to tune `RuntimeEnv` to avoid this issue from the tests? I do not like adding this addition to the runtime too much with the aim of avoiding migrations in testing. Future readers of this will not know why this is so, and we can be subject to unexpected behavior in the future. I would add a comment at least explain why this

Reply:
Another option is tweaking the block creation in `RuntimeEnv` somehow to skip the migration. Not sure if this is something easy to do

Reply:
Since this is my forced change which should not have been pushed to this branch in the first place, I will just revert it. However, it improves integration test time because it blocks migrations for each node we spawn. 

What unexpected behaviour are you fearing here @lemunozm? I think it is unexpected that migrations run for integration tests which is fixed this way. It's not the nicest solution but it does the job quickly without imposing any risk.

Reply:
Fixed in variant_to_local_rate_set

Reply:
> However, it improves integration test time because it blocks migrations for each node we spawn.

Good point!

> What unexpected behaviour are you fearing here @lemunozm?

Could some uses of the node underlying it exist that compile the runtime as `std` and where do we  expect some kind of migration, maybe some migration testing or so? Probably not now but it scared me 🙏🏻